### DISCUSSION TITLE
Feature Request: render code snippet for github.dev permalink

### DISCUSSION THREAD

**Discussion by sarisia** [community/community#38893] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

The permalink gotten from github web is prefixed with `https://github.com` and can be
previewed as a code snippet in issue comment, but one gotten from github.dev editor is
prefixed with `https://github.dev` and cannot be previewed:

![image](https://user-images.githubusercontent.com/33576079/201431491-80944acc-41d0-483a-b3c7-ff111a2f5fea.png)

Hope the latter will be supported!

**Comment by jfullstackdev** [comment]
I just created a new Issue on `github.dev` and there is the preview

![image](https://user-images.githubusercontent.com/47092464/201453543-7d886e5b-3cd8-4d45-8b5a-9ebd65111c93.png)
